src/startup.jl

```diff
@@ -44,7 +44,13 @@ if !symbols_present
     isfile(depfile) || error("PyCall not properly installed. Please run Pkg.build(\"PyCall\")")
     include(depfile) # generated by Pkg.build("PyCall")
     # Only to be used at top-level - pointer will be invalid after reload
-    libpy_handle = Libdl.dlopen(libpython, Libdl.RTLD_LAZY|Libdl.RTLD_DEEPBIND|Libdl.RTLD_GLOBAL)
+    try 
+		Libdl.dlopen(libpython, Libdl.RTLD_LAZY|Libdl.RTLD_DEEPBIND|Libdl.RTLD_GLOBAL)
+	catch err
+		error(err.msg, ". Please run `Pkg.build(\"PyCall\")` if your Python build has changed")
```

Comment:
This will fail if `err` doesn't have a `msg` field.  Maybe
```jl
if err isa ErrorException
    error(err.msg, ...)
else
   rethrow(err)
end
```

Reply:
Thanks @stevengj ! Added this if in the next commit...